Classes/Settings.cs

```diff
@@ -369,6 +330,36 @@ internal static int RssServerNumRecentEps
             }
         }
 
+        internal static string SetSaveFolder(Model.Programme progInfo)
```

Comment:
I realise they aren't properties due to the parameter but I think they should work like properties.  I agree that an overload of [] isn't a good fit.